NAMESPACE

```diff
@@ -32,6 +32,7 @@ export(crps_sample)
 export(dispersion)
 export(dss_sample)
 export(get_duplicate_forecasts)
+export(interval_coverage_deviation_quantile)
```

Comment:
I mean, we could have _all_ functions return a data.frame and then merging things together in `score()`. 
Or a wrapper in `score()` that checks whether the output is a data.frame, list or a vector? I'd be open to that, I just haven't thought about it long enough to have a good vision for how it would look like. 
I created an issue, #455. Could you please elaborate a bit on how you'd imagine this?